snippets/python/src/list_payments.py

```diff
@@ -0,0 +1,27 @@
+from breez_sdk_liquid import BindingLiquidSdk, ListPaymentsRequest, PaymentType
+import logging
+
+
+def list_payments(sdk: BindingLiquidSdk):
+    try:
+        # ANCHOR: list-payments
+        sdk.list_payments(ListPaymentsRequest([PaymentType.SEND]))
```

Comment:
In the other languages, this snippet shows a plain `list_payments` request.

For consistency, would it make sense to remove the `PaymentType.SEND` arg?

Reply:
Done [b6ead03](https://github.com/breez/breez-sdk-liquid-docs/pull/29/commits/b6ead03ddb6586bf5439ecf56a499b9ae7a91041)